### PR TITLE
Update version numbers to match PyPI

### PR DESCRIPTION
**Pull request by bsipocz**
@mwcraig - I guess the only problematic point is `gwcs` that just got on PyPI very recently. All the others are trivial updates of the version numbers to match the latest releases on PyPI (with the exception of ginga, where I wasn't sure it's worth updating to the ever changing newest).

### CODE REVIEW

**Comment by mwcraig** [comment]
@bsipocz -- thanks very much for this! A recipe template will need to be created for asdf to handle the numpy dependency. I can do a PR on your PR to do that tonight or you can create it if you want. 

I usually generate a recipe template with `conda skeleton pypi --setup-args="--offline" asdf` 

~~The only hassle here is that you need to use the version of conda-build I've bundled as a submodule because my PR to add the `--setup-args` option has not been merged (yet).~~

Actually, it is easier than I thought: if you have `conda-affiliate-builder` cloned on your machine, then `python affiliate_builder/prepare_packages.py requirements.yml` should generate recipes for you, which you could then use to make a recipe template.

File: requirements.yml [hunk 1]
```diff
@@ -108,11 +108,15 @@
...
-#   numpy_compiled_extensions: true
+- name: gwcs
+  version: 0.5
+  astropy_helpers: true
+  numpy_compiled_extensions: false
+
+- name: asdf
+  version: 1.0.0
+  astropy_helpers: true
+  numpy_compiled_extensions: yes
```
Comment (mwcraig) [inline requirements.yml]: `numpy_compiled_extensions` should be `false` for `asdf` because it is a pure python package.

**Comment by mwcraig** [comment]
@bsipocz -- this _should_ build once you merge the PR I just made. Do you want to do the asdf fix to `install_requires` or shall I?

File: requirements.yml (hunk 1 above)
Comment (bsipocz) [inline requirements.yml]: yes sorry, I was just a desperate to try to make it work (and don't know much about how the package building is working).

**Comment by bsipocz** [comment]
@mwcraig - Thanks. I already opened a PR for pyasdf https://github.com/spacetelescope/pyasdf/pull/185, and this should be fixed in its next release. Should I open an issue as a reminder that the recipe can be removed with the new release?

**Comment by mwcraig** [comment]
@bsipocz -- yes please!

**Comment by mwcraig** [comment]
@bsipocz -- looks like gwcs depends on asdf but doesn't list the requirement in setup.py. Same division of labor as for asdf, if that works for you...(I'll add the recipe template, you patch the original repo/make a reminder issue).

**Comment by bsipocz** [comment]
Ohh, great. Sorry for the unplanned messing with these.

**Comment by bsipocz** [comment]
Thanks @mwcraig. I vaguely remember that either of them has some version requirement for the latest (1.1) astropy, but probably it will be fine as is now.

**Comment by mwcraig** [comment]
Next up: `pydl`. Issue is a default path in a script that relies on the environment variable `HOME` being set...which is not the case on Windows. Just googled the fix, so I'll add the recipe template and do the PR on pydl.

**Comment by mwcraig** [comment]
@bsipocz -- now an `asdf` failure on python 3, because it tries to import a compatibility package that, I'm guessing, isn't installed on py3k...will come back to this on Saturday.

**Comment by bsipocz** [comment]
Thanks @mwcraig for all your inputs, and I'm really sorry that this PR (that I thought would be totally harmless) required that much work from you.

**Comment by mwcraig** [comment]
:tada: :confetti_ball: :balloon: :smile:

No problem, thanks for all of your work on it. The issues we ran into were coming sooner or later, so I'm happy to have gotten them out of the way now!

**Pull request merged**